website/docs/update-patterns.md

```diff
@@ -35,8 +35,19 @@ const deletedTodosObj = produce(todosObj, draft => {
 const updatedTodosObj = produce(todosObj, draft => {
 	draft["id1"].done = true
 })
+
+// replace & update in bulk
+const updatedTodosObj = produce(todosObj, draft => {
+	Object.assign(draft, {
+		id1: {done: true, body: "Take out the trash"},
+		id2: {done: true, body: "Check Email"},
+		id3: {done: true, body: "Feed my cat"}
+	})
+})
 ```
 
+Please keep in mind that when updating (nested) objects you should do so through mutation. Creating a new object and directly assigning it to a property in your immer draft object, will not work. This creates a new reference and does not update the existing object.
```

Comment:
:hand: Not sure this phrasing is right.

Immer ultimately tracks mutations to the draft and returns a new reference if anything really did change.

`draft.someField = {}` _is_ a mutation, and it _will_ end up returning a new result reference.  This is good.

Reply:
Discussed with Mark in private, replaced the note